guides/doc-Release_Notes/topics/foreman-3.4.0.adoc

```diff
@@ -0,0 +1,210 @@
+
+* Ubuntu 20.04.3 new installation media path - https://projects.theforeman.org/issues/34942[#34942]
+* Ubuntu Autoinstall hostname is not set correctly before finish template execution - https://projects.theforeman.org/issues/34908[#34908]
+* as a user, I would like to create a new host from the hostgroup page - https://projects.theforeman.org/issues/9770[#9770]
+
+=== Reporting
+
+* dont re-evaluate TemplateInputs on every loop iterationt re-evaluate TemplateInputs on every loop iteration - https://projects.theforeman.org/issues/34938[#34938]
+
+=== Facts
+
+* Details tab - System properties card updates - https://projects.theforeman.org/issues/34901[#34901]
+
+=== Unattended installations
+
+* userdata controller should be able to look up host by MAC address - https://projects.theforeman.org/issues/34887[#34887]
+
+=== Organizations and Locations
+
+* It is possible to create taxonomies named Any Organization' and 'Any Location'Any Organization and 'Any Location' and Any Location'Any Location - https://projects.theforeman.org/issues/28810[#28810]
+
+== Installer
+
+* /etc/pki/katello/certs/katello-server-ca.crt deployed with too strict permissions - https://projects.theforeman.org/issues/35335[#35335]
+* Move puppet certs exist hook to hooks/ - https://projects.theforeman.org/issues/35208[#35208]
+* Remove \`--disable-system-checks\` option - https://projects.theforeman.org/issues/35149[#35149]
+* Drop umask check from Katello hooks - https://projects.theforeman.org/issues/35138[#35138]
+* Introduce a development tuning profile - https://projects.theforeman.org/issues/35134[#35134]
+* Remove VM overcommit hook - https://projects.theforeman.org/issues/35133[#35133]
+* Starting or Restarting foreman.socket will raise a harmless "TCP_NODELAY failed: Operation not supported" error in Red Hat Satellite 6.9/6.10/6.11
+ - https://projects.theforeman.org/issues/35089[#35089]
+* Drop use of pulp_client certificates - https://projects.theforeman.org/issues/35005[#35005]
+* Drop apipie_dsl:cache generation - https://projects.theforeman.org/issues/34977[#34977]
+* Allow configuring cockpit with multiple origins through satellite-installer - https://projects.theforeman.org/issues/34943[#34943]
+* Option --certs-regenerate is stored and thus causes certificate regeneration to happen every run - https://projects.theforeman.org/issues/34875[#34875]
+* Missing smart-proxy-salt parameters in foreman-proxy scenario - https://projects.theforeman.org/issues/31502[#31502]
+
+=== Foreman modules
+
+* Insights client traffic through a Satellite 6.11 Capsule fails - https://projects.theforeman.org/issues/35278[#35278]
+* Autoindexing on /pub is broken because apache mod_autoindex is not loaded - https://projects.theforeman.org/issues/35227[#35227]
+
+=== External modules
+
+* cache-enabled setting for proxy content module isnt migrated to 'true' properlyt migrated to true' properlytrue properly properly - https://projects.theforeman.org/issues/35157[#35157]
```

Comment:
I'm not sure what went wrong here. I do recall I had some troubles in the past with references in the script. Perhaps they're surfacing again.

Reply:
yeah, there were few more, fixed those